src/Components/AbilityTree/AbilityTree.java

```diff
@@ -1,83 +0,0 @@
-package Components.AbilityTree;
-
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
-
-public class AbilityTree {
-
-    // A map that stores abilities and their prerequisites
```

Comment:
This map assumes linear progression. I think that's fine! If you want to have multiple prerequisites (i.e., need fireball and ice lance), then you might need to rethink this design a bit.